File: vm_Assignments/vmc/L2.6/ComputeChange.c
#include<stdio.h>
void compute_change(int m)
{
    int n = 100 - m;
    int quarter = n/25;
    int dime = ((n % 25)/10);
    int nickel = (((n % 25)%10)/5);
    int pennis = n -(25*quarter + 10*dime + 5*nickel);
    printf("%d quarters\n", quarter);
    printf("%d dime\n", dime);
    printf("%d nickel\n", nickel);
    printf("%d pennis\n", pennis);
}


int main()
{
    compute_change(8);
}

File: vm_Assignments/vm/Beginning/L2.6/ComputeChange.py
def compute_change(m):
    n = 100 - m
    quarter = int(n/25)
    dime = int((n % 25)/10)
    nickel = int(((n % 25) % 10)/5)
    pennies = n - (25*(quarter)+(10*dime)+(5*nickel))
    print(quarter,"quarters")
    print(dime,"dimes")
    print(nickel,"nickels")
    print(pennies, "pennies")
compute_change(8)